okta-hosted-login/src/OidcCompliant.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { useOktaAuth } from '@okta/okta-react';
+
+const OidcCompliant = () => {
+  const { oktaAuth, authState } = useOktaAuth();
+
+  const login = async () => oktaAuth.signInWithRedirect('/');
+
+  const searchParams = new URLSearchParams(useLocation().search);
+  const iss = searchParams.get('iss');
+  // if app can be configured for multiple issuers use this value
```

Comment:
what do you mean here? I'm trying to learn from your example. In testing with an idp initiated login from their dashboard to our app setting oktaAuth.options.issuer to the issuer there results in a 400. I'm guessing I have a misconfiguration, just not sure what currently.